docs/README.md

```diff
@@ -86,7 +86,7 @@ description: Starship is the minimal, blazing fast, and extremely customizable p
 
    #### Powershell
 
-   Add the following to the end of `~\Documents\PowerShell\Microsoft.PowerShell_profile.ps1` (or `~/.config/powershell/Microsoft.PowerShell_profile.ps1` on -Nix):
+   Add the following to the end of `Microsoft.PowerShell_profile.ps1`. You can check the location of this ps1 file with `$PROFILE` command in PowerShell (typically the path is `~\Documents\PowerShell\Microsoft.PowerShell_profile.ps1` or `~\.config\powershell\Microsoft.PowerShell_profile.ps1` on -Nix)
```

Comment:
I think this would work better without the brackets:
```suggestion
   Add the following to the end of `Microsoft.PowerShell_profile.ps1`. You can check the location of this ps1 file with `$PROFILE` command in PowerShell. Typically the path is `~\Documents\PowerShell\Microsoft.PowerShell_profile.ps1` or `~\.config\powershell\Microsoft.PowerShell_profile.ps1` on -Nix.
```


Also I'd either say "by querying the `$PROFILE` variable" instead of "`$PROFILE` command " or change the command to `Write-Host $PROFILE`.

Instead of "this ps1 file" I'd also prefer "this file".

Reply:
@davidkna Thank you for your review. I reflected opinions.